src/component/Nav/NavModal.js

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { USER_INFO } from '../../pages/Redirect/Redirect';
+
+import styled from 'styled-components';
+
+export const NavModal = ({ handleModal }) => {
+  const [toggle, settoggle] = useState(false);
+  const [grade, setgrade] = useState('SILVER');
+
+  const userInfo = JSON.parse(localStorage.getItem(USER_INFO));
+
+  const onOff = () => {
+    settoggle(!toggle);
+    gradeRequest();
+  };
+
+  const gradeRequest = () => {
+    fetch('http://192.168.43.114:8000/users/level', {
+      method: 'PATCH',
+      headers: {
+        Authorization: userInfo.token,
+      },
+      body: JSON.stringify({
+        agreement: toggle ? 'False' : 'True',
+        userlevel: toggle ? 1 : 2,
+      }),
+    })
+      .then(res => res.json())
+      .then(res => {
+        if (res.id === 2) {
+          setgrade('GOLD');
+        } else if (res.id === 1) {
+          setgrade('SILVER');
+        }
+      });
+  };
+
+  return (
+    <Modal>
+      <Name>
+        {userInfo ? `${userInfo.name}님` : '로그인 해주세요.'}
+        {userInfo && grade && <Grade>{grade}</Grade>}
+        <br />
+        <Xmark>
+          <i className="fas fa-times" onClick={handleModal}></i>
+        </Xmark>
+        <Button>내 정보 수정</Button>
+        <Icon>
+          <i className="far fa-handshake">약관동의</i>
+          {toggle ? (
+            <Toggle onClick={onOff}>ON</Toggle>
+          ) : (
+            <Toggle onClick={onOff}>OFF</Toggle>
+          )}
```

Comment:
```suggestion
          {toggle ? (
            <Toggle onClick={onOff}>{toggle ? "ON" : "OFF"}</Toggle>
```
이렇게 표현 가능해보입니다! 중복되는 부분과 변하는 부분이 뭐가 있는지 잘 판단후에 적용해보세요!

Reply:
넵 감사합니다.